### PR TITLE
Patch soljson.js to provide backwards compatibility with older emscripten versions.

### PR DESCRIPTION
**Pull request by chriseth**
Fixes https://github.com/ethereum/solidity/issues/5830

### CODE REVIEW

File: scripts/travis-emscripten/build_emscripten.sh [hunk 1]
```diff
@@ -96,6 +96,8 @@ make -j 4
 
 cd ..
 mkdir -p upload
+# Patch soljson.js to provide backwards-compatibility with older emscripten versions
+echo ";/* backwards compatibility */ Module['Runtime'] = Module;" >> build/libsolc/soljson.js
```
Comment (ekpyron) [inline scripts/travis-emscripten/build_emscripten.sh:100]: Do we need the ``;`` at the end? Resp. is it even valid to have it?

Comment (chriseth) [reply scripts/travis-emscripten/build_emscripten.sh:100]: works in node

**Comment by chriseth** [comment]
Tested with solc-js 0.5.0, all tests are green apart from the version test (as expected).

**Comment by axic** [comment]
Wait a second. How did Zeppelin tests finish *before this PR* if they use truffle and an old solc-js?

**Comment by axic** [comment]
Because we patch solc to use solc master: https://github.com/ethereum/solidity/blob/develop/test/externalTests.sh#L64

If we are serious about #5830 then we should always patch it to 0.5.0.

**Comment by chriseth** [comment]
I think this was needed while we were preparing for 0.5.0, we might want to revert the other patch now - or run with both versions.

Anyway, is this PR here good to go?

**Comment by axic** [comment]
I'd prefer if this could change the Zeppelin test to use solc-js 0.5.0 so that we can confirmation it works and sets the stage for future changes adhering to this rule.

**Comment by chriseth** [comment]
I'll merge this now so that we get a nightly build with it and create a new PR that also changes zeppelin.

**Pull request merged**